plugins/notify.py

```diff
@@ -0,0 +1,11 @@
+from will.plugin import WillPlugin
+from will.decorators import respond_to, periodic, hear, randomly, route, rendered_template
+
+
+class NotifyPlugin(WillPlugin):
+
+    @route("/notify/<build_id>/<text>")
+    def send_notification(self, build_id, text):
+        notification_list = self.load("notify_{}".format(build_id), '')
+        self.say("{} BuildID: {}, Message: {}".format(notification_list, build_id, text)) 
```

Comment:
This puts the output in the default channel.  Could use some magic to recover the message to use reply_to instead of say.